src/hotspot/share/gc/shenandoah/shenandoahFullGC.cpp

```diff
@@ -322,6 +322,10 @@ void ShenandoahFullGC::do_it(GCCause::Cause gc_cause) {
     phase3_update_references();
 
     phase4_compact_objects(worker_slices);
+
```

Comment:
This is the part of #235, is it? Should not be in this PR, I think.

Reply:
Aha! I think you can merge from master and this would be gone. I am surprised GH does not complain about this, but I guess as long as affected hunks are the same, there is no merge conflict.